.devops/templates/runpublishvrscreenshot.yml

```diff
@@ -46,6 +46,7 @@ steps:
   - script: |
```

Comment:
## [🕵 fluentuiv9 [New] Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=undefined&project=fabricpublic&prId=27462&commitId=a7c817815f468bb58f293c2095525af051cf78c6&env=prod)

✅ undefined


      <details><summary><b>undefined</b> 114 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|
|undefined|NaN| Changed|

</details>


<div id="vrtCommentE2E Tests-v9"/>